dev/Expander/Expander.xaml

```diff
@@ -23,14 +23,12 @@
         <Setter Property="Background" Value="{ThemeResource ExpanderBackground}" />
         <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />
         <Setter Property="MinWidth" Value="{ThemeResource FlyoutThemeMinWidth}" />
-        <Setter Property="MaxWidth" Value="{ThemeResource FlyoutThemeMaxWidth}" />
         <Setter Property="MinHeight" Value="{StaticResource ExpanderMinHeight}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ExpanderBorderThickness}" />
         <Setter Property="BorderBrush" Value="{ThemeResource ExpanderBorderBrush}" />
         <Setter Property="Padding" Value="{StaticResource ExpanderHeaderPadding}" />
-        <Setter Property="HorizontalAlignment" Value="Stretch" />
-        <Setter Property="HorizontalContentAlignment" Value="Left" />
-        <Setter Property="VerticalContentAlignment" Value="Center" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="VerticalAlignment" Value="Center" />
         <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
```

Comment:
The ControlTemplate itself should be pulled out as it's own named section. Makes it easier to reuse the same template for the control, but customize the style properties independently.

Reply:
Yea, I know you like to separate XAML out quite a bit into components within the toolkit. That's not historically what has been done in either WPF or UWP/WinUI afaik? Is that really the new convention WinUI is following now as well?

It certainly does make it easier to re-template and not worry about default property values. So I can for sure separate this out, just want to make sure that's how things are being done in WinUI.

Reply:
I should add the reason for the hesitation here: Adding a named template in another section would become something developers rely on over time. This means the name is essentially locked in and can't be modified in the future. I'm not sure if WinUI wants to accept that commitment.

Reply:
I mean relying on the name is no different than all the style resources for other things are now. And we have to start somewhere... 🙂

Reply:
Ok, marking as resolved then.